src/_locales/en_US/messages.json

```diff
@@ -135,6 +135,10 @@
         "message": "Replace social widgets",
         "description": "Checkbox label on the general settings page"
     },
+    "options_spoof_referrer_checkbox": {
+        "message": "Spoof the Referrer header",
```

Comment:
It would be useful to have this text link out to an explanation of what a "referer" header is. Also, should we spell it the English way, or the correct way? https://en.wikipedia.org/wiki/HTTP_referer